client/asset/eth/nodeclient.go

```diff
@@ -69,10 +69,11 @@ type nodeClient struct {
 }
 
 func newNodeClient(dir string, net dex.Network, log dex.Logger) (*nodeClient, error) {
+	lggr := dex.StdOutLogger("node", dex.LevelError)
 	node, err := prepareNode(&nodeConfig{
 		net:    net,
 		appDir: dir,
-		logger: log.SubLogger("NODE"),
+		logger: lggr,
```

Comment:
The eth node is really loud. This is not correct, but ok if we set it to error by default?

Reply:
Using "ITNL" for internal as a key. Will add something to the dexc config options I guess. We could just accept the level map string which would look like "key=lvl,key=lvl,...."

Reply:
Added the beginnings of a way to control this per logger.